editor/selectors.js

```diff
@@ -3,6 +3,7 @@
  */
 import moment from 'moment';
 import { first, last, get } from 'lodash';
+import { createSelector } from 'reselect';
```

Comment:
>Though according to their docs, we could write this

Is this something you created, or in their docs? How is it used? Isn't creating and calling `selector` in the inner function expensive?